frontend/test/pytest/test_device_api.py

```diff
@@ -58,6 +58,10 @@ def preprocess(self, execution_config: ExecutionConfig = DefaultExecutionConfig)
         return transform_program, execution_config
 
 
+@pytest.mark.skipif(
+    not pathlib.Path(get_lib_path("runtime", "RUNTIME_LIB_DIR") + "/libdummy_device.so").is_file(),
+    reason="lib_dummydevice.so was not found.",
+)
```

Comment:
Should we allow these tests to be skipped? In what situation would the library be unavailable?

Reply:
Perfect, thanks :)